test/fail_compilation/fail16206b.d

```diff
@@ -0,0 +1,12 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail16206b.d(12): Error: expected 2 arguments for `hasMember` but had 3
+---
+*/
+
+struct S {
```

Comment:
Curly braces on their own lines. Applies to all test files and the changelog entry.

Reply:
OT:
Btw I am not sure whether we need to be so picky for the test files, because it mostly doesn't matter there.
No one looks actively at the tests anyhow, but only when they fail.

Reply:
BTW, I think that tests should be treated just as the rest of the code. Same standards, style, refactoring and so on.